### PR TITLE
Measure DB accuracy with graffiti guess

### PR DESCRIPTION
**Pull request by michaelsproul**
* Add a `graffiti_guess` column to the database
* Estimate the confusion matrix for each client by comparing classifier results against the graffiti guess
* When running the API server, disable the (slow) classifier load if the `DISABLE_CLASSIFIER` environment variable is set

I think this is quite prone to overfitting, as in my most recent training data set there are very few misclassifications, even for Nimbus and Lodestar (very suspicious)

```
curl "http://localhost:8000/confusion/Lighthouse/0/10000000"
{"true_pos": 21925, "true_neg": 7931, "false_pos": 16, "false_neg": 0}                                                                                                                                                         

curl "http://localhost:8000/confusion/Prysm/0/10000000"
{"true_pos": 279, "true_neg": 29584, "false_pos": 0, "false_neg": 9}

curl "http://localhost:8000/confusion/Teku/0/10000000"
{"true_pos": 6684, "true_neg": 23188, "false_pos": 0, "false_neg": 0}

curl "http://localhost:8000/confusion/Nimbus/0/10000000"
{"true_pos": 928, "true_neg": 28944, "false_pos": 0, "false_neg": 0}

curl "http://localhost:8000/confusion/Lodestar/0/10000000"
{"true_pos": 40, "true_neg": 29825, "false_pos": 0, "false_neg": 7}
```

### CODE REVIEW

**Comment by ariskk** [comment]
Ah shit I said I would open a PR with that, didn't I. LGTM!

**Comment by michaelsproul** [comment]
Sorry to front-run you, I got curious!

I'm looking forward to seeing the comparison once the DB ingests some blocks from outside the training set
